apps/backend/src/main/java/ch/heig/pdg/backend/services/CategoryService.java

```diff
@@ -13,26 +13,56 @@
 import java.util.stream.Collectors;
 
 @Service
-public class CategoryService {
+public class CategoryService extends AbstractService {
     private final CategoryRepository categoryRepository;
-    private final InventoryRepository inventoryRepository;
     private final CategoryMapper categoryMapper;
 
     public CategoryService(CategoryRepository categoryRepository, InventoryRepository inventoryRepository, CategoryMapper categoryMapper) {
+        super(inventoryRepository);
         this.categoryRepository = categoryRepository;
-        this.inventoryRepository = inventoryRepository;
         this.categoryMapper = categoryMapper;
     }
 
+    public CategoryDTO getCategory(Integer id) {
+        return (CategoryDTO) this.categoryMapper.getDTO(
```

Comment:
Did you randomly press `Enter` on your keyboard ?

Reply:
yes